docs/deployment/settings.md

```diff
@@ -455,6 +455,7 @@ You can configure the Kyuubi properties in `$KYUUBI_HOME/conf/kyuubi-defaults.co
 | kyuubi.server.batch.limit.connections.per.user           | &lt;undefined&gt; | Maximum kyuubi server batch connections per user. Any user exceeding this limit will not be allowed to connect.                                                                                                                                | int      | 1.7.0 |
 | kyuubi.server.batch.limit.connections.per.user.ipaddress | &lt;undefined&gt; | Maximum kyuubi server batch connections per user:ipaddress combination. Any user-ipaddress exceeding this limit will not be allowed to connect.                                                                                                | int      | 1.7.0 |
 | kyuubi.server.info.provider                              | ENGINE            | The server information provider name, some clients may rely on this information to check the server compatibilities and functionalities. <li>SERVER: Return Kyuubi server information.</li> <li>ENGINE: Return Kyuubi engine information.</li> | string   | 1.6.1 |
+| kyuubi.server.limit.client.fetch.max.rows                | 5000              | Max rows limit for getting result row set operation. If the max rows specified by client-side is larger than the limit, request will fail directly.                                                                                            | int      | 1.7.0 |
```

Comment:
```
kyuubi.server.limit.connections.per.user.ipaddress
kyuubi.server.batch.limit.connections.per.user.ipaddress
kyuubi.server.limit.client.fetch.max.rows   
```


should we rename the config `kyuubi.server.batch.limit.connections.per.user.ipaddress` to `kyuubi.server.limit.batch.connections.per.user.ipaddress` ? @pan3793

Reply:
raised #4398